srcpkgs/zssh/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'zssh'
+pkgname=zssh
+version=1.5c
+revision=1
+build_style=gnu-configure
+makedepends="readline-devel"
+depends="openssh inetutils-telnet lrzsz"
+short_desc="Zmodem aware telnet and ssh cli client"
+maintainer="Niels Haedecke <webmaster@ufud.org>"
+license="GPL-2.0-only"
+homepage="https://zssh.sourceforge.net/"
+distfiles="http://prdownloads.sf.net/zssh/zssh-${version}.tgz"
```

Comment:
```suggestion
distfiles="${SOURCEFORGE_SITE}/zssh/zssh-${version}.tgz"
```

Reply:
You will have to Google git basics

Reply:
@paper42 done - consolidated commit history. Is this okay now?

Reply:
fixed commit message to conform to CONTRIBUTING.md